packages/jaeger-ui/src/components/TracePage/TracePageHeader.js

```diff
@@ -30,14 +29,18 @@ import { formatDatetime, formatDuration } from '../../utils/date';
 import prefixUrl from '../../utils/prefix-url';
 
 import './TracePageHeader.css';
+import TracePageSearchBar from './TracePageSearchBar';
```

Comment:
Please move this import to fall after TracePageHeader.track.

Imports are sorted alphabetically with more dots under less dots and css files that are imported for the side effect go below the other imports. Also, external packages go above local resources, separated by a blank line, with react going above others.

	import * as React from 'react';
	import extA from 'a';
	import extB from 'b';

	import a from './a';
	import z from './z';
	import b from '../b';

	import 'a.css';

Reply:
oops, forgot to push my commit